src/main/java/org/tugraz/sysds/api/DMLScript.java

```diff
@@ -83,7 +83,6 @@
 {
 	private static ExecMode   EXEC_MODE          = DMLOptions.defaultOptions.execMode;    // the execution mode
 	public static boolean     STATISTICS          = DMLOptions.defaultOptions.stats;       // whether to print statistics
-	public static boolean     FINEGRAINED_STATISTICS  = false;                             // whether to print fine-grained statistics
```

Comment:
Please don't push other changes from master to your PR unless you squash everything together as this makes it harder to review the actual changes.

Reply:
I will try to keep that in mind in the future. I think right now I can not fix this anymore without creating a new PR right?